ci_config.json

```diff
@@ -1,10 +1,10 @@
 {
   "_comment": ["This is intentionally undocumented in README.md, do not use it.",
                "Using it is an automatic CI error.",
-               "skip_linux and skip_windows are lists of wraps that are known",
+               "broken_linux, broken_windows and broken_macos are lists of wraps that are known",
                "to be broken on the CI and ***MUST*** get repaired before updates",
-               "will be accepted. If upstream does not support Windows, set that",
-               "wrap to linux_only instead.",
+               "will be accepted. If upstream does not support an Operating System, set that",
+               "explicitly disable that Operating System in build_on.",
```

Comment:
"set that explicitly disable that Operating System"

The "set that" part is left over from the previous wording.